dependencies.yaml

```diff
@@ -46,19 +46,28 @@ dependencies:
           - matrix:
               cuda: "11.2"
             packages:
-              - cudatoolkit=11.2
+              - cuda-version=11.2
+              - cudatoolkit
           - matrix:
               cuda: "11.4"
             packages:
-              - cudatoolkit=11.4
+              - cuda-version=11.4
+              - cudatoolkit
           - matrix:
               cuda: "11.5"
             packages:
-              - cudatoolkit=11.5
+              - cuda-version=11.5
+              - cudatoolkit
           - matrix:
               cuda: "11.8"
             packages:
-              - cudatoolkit=11.8
+              - cuda-version=11.8
+              - cudatoolkit
+          - matrix:
+              cuda: "12.0"
+            packages:
+              - cuda-version=12.0
+              - cuda-nvcc
```

Comment:
```suggestion
              - cuda-nvrtc
```

Reply:
From what I understand of @gmarkall 's suggestion we don't need `cuda-nvrtc` given we're not doing any linkage whatsoever in Dask-CUDA, thus I think we should have only `cuda-nvcc` here.

Reply:
Ok will leave this here and we can revisit as needed